Libraries/LibWeb/HTML/HTMLOptionElement.cpp

```diff
@@ -147,6 +154,8 @@ String HTMLOptionElement::text() const
 void HTMLOptionElement::set_text(String const& text)
 {
     string_replace_all(text);
+
+    update_selection_label();
```

Comment:
Does the call to `string_replace_all()` here not already cause `children_changed()` to be called?

Reply:
Ah true, and the `set_attribute` call in `set_label` also triggers `attribute_changed`